runtime/runtime-params-estimator/src/vm_estimator.rs

```diff
@@ -112,15 +112,16 @@ fn compile(code: &[u8], gas_metric: GasMetric, vm_kind: VMKind) -> (f64, f64) {
     (code.len() as f64, end / (NUM_ITERATIONS as f64))
 }
 
-/// Cost of the most CPU demanding operation.
+/// Cost of the compile contract with vm_kind
 pub fn cost_to_compile(gas_metric: GasMetric, vm_kind: VMKind) -> (Ratio<u64>, u64) {
-    // Call once for the warmup.
+    println!("About to compile contracts.....");
     let (sx, sy) =
         compile(include_bytes!("../test-contract/res/small_contract.wasm"), gas_metric, vm_kind);
     let (mx, my) =
         compile(include_bytes!("../test-contract/res/medium_contract.wasm"), gas_metric, vm_kind);
```

Comment:
As a medium contract you can grab https://github.com/near/near-sdk-rs/blob/master/examples/status-message-collections/res/status_message_collections.wasm as it's `116Kb`